src/Installer/redist-installer/targets/GenerateLayout.targets

```diff
@@ -590,13 +590,6 @@
              Importance="High" />
   </Target>
 
-  <Target Name="GeneratePrecomputedRarCache" DependsOnTargets="LayoutTemplates;LayoutManifests;LayoutBundledTools">
```

Comment:
Per Triage: let's leave the target definition in just in case someone comes back and asks us about this, we can find it more easily.